### ISSUE TITLE
Cairo tests fail on 32-bit Appveyor

### ISSUE THREAD

**Issue by simonbyrne** [JuliaLang/BinDeps.jl#356]
Apparently have been broken for 9 months https://ci.appveyor.com/project/nalimilan/bindeps-jl/build/1.0.222

The first failure was on #300: that doesn't seem related, so it was probably a change with Cairo.jl (which doesn't have 32-bit tests enabled).

Either we should fix Cairo.jl, or remove it from the tests.

**Comment by simonbyrne** [comment]
Looks like some gcc issue: https://github.com/JuliaGraphics/Cairo.jl/pull/196

The easiest option at this stage is to move Cairo to binaryprovider.jl, and get rid of the test here.

**Comment by tkelman** [comment]
The point of the test is to exercise code paths here, so if the upstream package changes to no longer use BinDeps then the test dependency should be upper-bounded.
Cairo can probably be skipped on win32 for now but it should be replaced by a different WinRPM-using package that does work.

**Comment by lobingera** [comment]
@simonbyrne Do you have experiences to move libraries to binaryprovider?

**Comment by simonbyrne** [comment]
No, unfortunately. Ask on the JuliaLang Slack #bindeps2 channel

**Comment by lobingera** [comment]
> easiest option
?

**Comment by simonbyrne** [comment]
Well, I don't understand BinDeps.jl either. 😄
